cellfinder_napari/detect.py

```diff
@@ -63,15 +63,15 @@ def widget(
         soma_spread_factor: float,
         max_cluster_size: int,
         classification_options,
-        trained_model: Path,
+        trained_model: Optional[Path],
         misc_options,
         start_plane: int,
         end_plane: int,
         n_free_cpus: int,
         analyse_local: bool,
         debug: bool,
         reset_button,
-    ) -> List[napari.types.LayerDataTuple]:
+    ) -> None:
```

Comment:
This appears to have been here prior to #54, but the function doesn't actually return anything

Reply:
That's right - good catch.

From looking at the git blame, the return type changed [in a refactoring commit](https://github.com/brainglobe/cellfinder-napari/commit/1ba0ac6abd9028e57e0d659e9d243849ebc96851) but the type hint wasn't adapted. Before, the two points layers were returned - After, they were added directly to the viewer in the `add_layers` callback function.